src/measure_theory/lp_space.lean

```diff
@@ -413,6 +413,38 @@ lemma mem_ℒp.of_bound [finite_measure μ] {f : α → E} (hf : ae_measurable f
   mem_ℒp f p μ :=
 (mem_ℒp_const C).of_le hf (hfC.mono (λ x hx, le_trans hx (le_abs_self _)))
 
+lemma snorm'_mono_measure {μ ν : measure α} {f : α → F} (hμν : ν ≤ μ) (hq : 0 ≤ q) :
```

Comment:
Should be `f` explicit here, and in the next two lemmas?  (I am not sure ... in your use case here it is not needed.)

Reply:
Thanks. I made it explicit, since it cannot be inferred from other arguments. It was not needed in the places where I use those lemmas, but I think it's better to have one more argument than to require `@` when one want to write `have h := snorm'_mono_measure ...`